src/components/Clock.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    timer: new Date(),
+  };
+
+  timerId:NodeJS.Timeout = setInterval(() => {}, 1000);
```

Comment:
Краще не створювати зайвий інтервал
```suggestion
  timerId?: NodeJS.Timeout;
```